### ISSUE TITLE
Error extracting High Res textures

### ISSUE THREAD

**Issue by 09williamsad** [mhvuze/WorldChunkTool#13]
A majority of the high res weapon textures are skipped due to an error.
[texture extract log.txt](https://github.com/mhvuze/WorldChunkTool/files/4087535/texture.extract.log.txt)

This is with version 1.2.1. 
Has not happend with the base game as far as I can tell.
Could have happened with more than just weapon textures but cmd history does not go back far enough to see.

See example below and attached log.
```
Error occured while extracting \wp\rod\rod009\mod\rod009_CMM.tex(chunkG0), skipped. Please try again later.
   at WorldChunkTool.Utils.GetChunkKey(Int32 i)
   at WorldChunkTool.ChunkOTF.getDecompressedChunk(Int64 offset, Int64 size, BinaryReader reader, Boolean FlagBaseGame, Int32 chunkNum)
   at WorldChunkTool.ChunkOTF.ExtractSelected(List`1 itemlist, String BaseLocation, Boolean FlagBaseGame)
```

**Comment by mhvuze** [comment]
Thanks for bringing this to my attention. The hi-res chunk (despite being considerably smaller) requires more decryption keys so to speak than the regular chunk. Should be an easy fix, I will try to get it done today (well, I will ask legendff to gen more keys and implement them if he got time today).

**Comment by mhvuze** [comment]
Fixed in [1.2.2](https://github.com/mhvuze/WorldChunkTool/releases/tag/1.2.2). Please note that the HiRes ChunkG0 requires a whopping 86GB of storage unpacked. The tool will still throw an error if you run out of disk space.
